pkg/core/resources/apis/mesh/dataplane_validator.go

```diff
@@ -50,7 +50,6 @@ func (d *DataplaneResource) Validate() error {
 
 		err.AddErrorAt(net.Field("gateway"), validateGateway(d.Spec.GetNetworking().GetGateway()))
 		err.Add(validateNetworking(d.Spec.GetNetworking()))
-		err.Add(validateProbes(d.Spec.GetProbes()))
```

Comment:
To me you still validate this. You only make it optional

Reply:
I forgot to bring it back when I restore the `Probes` field on the `Dataplane` object. I'll restore it now.